consensus/istanbul/backend/announce.go

```diff
@@ -349,6 +349,26 @@ func (sb *Backend) generateAndGossipAnnounce() error {
 		return nil
 	}
 
+	// Generate a new versioned enode message with the updated version
+	versionedEnodeMsg, err := sb.generateVersionedEnodeMsg(announceVersion)
+	if err != nil {
+		return err
+	}
+	if versionedEnodeMsg != nil {
+		sb.selfVersionedEnodeMsgMu.Lock()
```

Comment:
Would it be useful to define these three lines as a helper function?

Reply:
Good idea https://github.com/celo-org/celo-blockchain/pull/873/commits/97d7ef05988a92748c57bb556e6b723b7d891b40